quantum/rgb_matrix.c

```diff
@@ -424,6 +424,28 @@ __attribute__((weak)) void rgb_matrix_indicators_kb(void) {}
 
 __attribute__((weak)) void rgb_matrix_indicators_user(void) {}
 
+void rgb_matrix_indicators_advanced(effect_params_t *params) {
+    /* special handling is need "params->iter", since it's already been incremented.
```

Comment:
`special handling is need "params->iter",`

I think something is missing from here...